spec/models/searchworks_item_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe SearchworksItem do
+  let(:item_id) { SearchworksItem.new }
```

Comment:
If you want to use subject you can write:

```ruby
let(:subject) { SearchworksItem.new(item_id) }
```

You can then use subject in your assertions.  (e.g. `expect(subject.searchworks_uri).to eq('http://searchworks.stanford.edu')`)